src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1952,11 +1968,11 @@ public partial class RankException : System.SystemException
         [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
         public static System.ReadOnlySpan<T> DangerousCreate(object obj, ref T objectData, int length) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
-        [System.ObsoleteAttribute("Equals() on ReadOnlySpan will always throw an exception. Use == instead.")]
+        [System.ObsoleteAttribute("Equals() on Span will always throw an exception. Use == instead.")]
```

Comment:
The original message looks right. This looks like a bug in the implementation.

Reply:
This just updated to what is in the implementation. We could definitely update both to be consistent with ReadOnlySpan.

Reply:
Fix: https://github.com/dotnet/coreclr/pull/15820